sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1766,14 +1774,6 @@ object SQLConf {
       .checkValues(StoreAssignmentPolicy.values.map(_.toString))
       .createWithDefault(StoreAssignmentPolicy.ANSI.toString)
 
-  val ANSI_ENABLED = buildConf("spark.sql.ansi.enabled")
```

Comment:
I think we need to handle this as a deprecated synonym for the new config. While we can break this kind of thing in Spark 3, it's easy not to.

Reply:
Copy, done in d76d6a5